guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<p style="user-select: auto;">Formatting for tracker heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Adjusted it to match the formatting of the workforce heading</p>

Reply:
<p>Thanks for the input <a href="https://github.com/AtmaMani" target="_blank">AtmaMani</a>!</p>